File: frontend/fix_schedule.js
const fs = require('fs');
const filePath = 'frontend/src/pages/Schedule.jsx';
let content = fs.readFileSync(filePath, 'utf8');

// Remove the unclosed div
content = content.replace('<div className="calendar-section">', '');

// Wait, the style tag I injected looks like:
// <style>{`
//   @media print {
//     .print-calendar-only .print\\:hidden, .print-calendar-only .detailed-list-section { display: none !important; }

content = content.replace('.print-calendar-only .detailed-list-section { display: none !important; }', '.print-calendar-only .mt-12 { display: none !important; }');
content = content.replace('.print-list-only .calendar-section { display: none !important; }', '.print-list-only .mb-8, .print-list-only .grid-cols-7 { display: none !important; }');

fs.writeFileSync(filePath, content, 'utf8');
console.log('Fixed unclosed div and updated print css.');

File: frontend/fix_schedule.py
import os

filepath = 'frontend/src/pages/Schedule.jsx'
with open(filepath, 'r', encoding='utf8') as f:
    content = f.read()

content = content.replace('<div className="calendar-section">', '')
content = content.replace('.print-calendar-only .detailed-list-section { display: none !important; }', '.print-calendar-only .mt-12 { display: none !important; }')
content = content.replace('.print-list-only .calendar-section { display: none !important; }', '.print-list-only .mb-8, .print-list-only .bg-\\[\\#181818\\] { display: none !important; }')

with open(filepath, 'w', encoding='utf8') as f:
    f.write(content)

print("Fixed Schedule.jsx unclosed div")
